### PR TITLE
Added ufmean to the droplet_diag

### PR DESCRIPTION
**Pull request by drichte2**
Adds a few more items to droplet_diag

### CODE REVIEW

**Comment by sjsprecious** [comment]
The verification result on Gust with `namelist_ASD.verify` namelist is similar with that from PR #124.

# 2 MPI ranks (CPU, intel) versus 2 A100 GPU (nvhpc)

## Metrics

38 stat variables are identical
41 stat variables show a mean relative difference > 1e-06
7 stat variables show a mean relative difference <= 1e-06

## 0th-order

4 stat variables are identical
8 stat variables show a mean relative difference > 1e-06
0 stat variables show a mean relative difference <= 1e-06

## 1st-order

254 stat variables are identical
2 stat variables show a mean relative difference > 1e-06
0 stat variables show a mean relative difference <= 1e-06

File: src/droplet.F [hunk 1]
```diff
@@ -502,7 +502,7 @@ subroutine droplet_driver(dt,dbldt,mtime,xh,uh,ruh,xf,yh,vh,rvh,yf,zh,mh,rmh,zf,
     do k=kb,ke
        do j=jb,je
           do i=ib,ie
-             dpten(i,j,k,5) = dpten(i,j,k,5)*pi0(i,j,k)
+             dpten(i,j,k,1) = dpten(i,j,k,1)*pi0(i,j,k)
```
Comment (sjsprecious) [inline src/droplet.F:505]: Is this a bug fix or just due to a reorder of `dpten` index?

**Review by sjsprecious — COMMENTED** [review]
Thanks @drichte2 for issuing this PR. It looks good to me and the verification result looks reasonable, too. I just have one clarification question.

File: src/droplet.F (hunk 1 above)
Comment (drichte2) [reply src/droplet.F:505]: Bug fix! George pointed it out last week

Comment (sjsprecious) [reply src/droplet.F:505]: Thanks for the clarification.

**Review by sjsprecious — APPROVED** [review]

**Review by johnmauff — APPROVED** [review]
Looks good to me.

**Pull request merged**